src/odemis/driver/test/xt_client_test.py

```diff
@@ -571,15 +571,15 @@ def test_fwd_follows_z(self):
         # Set fwd follows z back to initial value
         self.microscope.set_fwd_follows_z(init_fwd_follows_z)
 
-    def test_pitch(self):
-        """Test setting and getting the pitch between two beams within the multiprobe pattern."""
-        pitch_range = self.microscope.pitch_info()["range"]
-        current_pitch = self.microscope.get_pitch()
-        new_pitch = current_pitch + 1
-        new_pitch = new_pitch if pitch_range[0] < new_pitch < pitch_range[1] else current_pitch - 1
-        self.microscope.set_pitch(new_pitch)
-        self.assertAlmostEqual(new_pitch, self.microscope.get_pitch())
-        self.microscope.set_pitch(current_pitch)
+    def test_delta_pitch(self):
+        """Test setting and getting the delta pitch between two beams within the multiprobe pattern."""
```

Comment:
This docstring now sounds a bit strange. I'm fine with just changing it to: 
`"""Test setting and getting the delta pitch."""`
As that is consistent with for instance test_stigmator and test_fwd_follows_z